packages/core/src/internal/clients/auth/AbstractAuthClient.ts

```diff
@@ -22,36 +22,55 @@ export abstract class AbstractAuthClient implements IAuthClient {
   }
 
   /**
-   * Generates a new auth token, along with a refresh token to refresh the auth token before expiry.
+   * Generates a new API key, along with a refresh token to refresh the API key before expiry.
    *
-   * @param {TokenScope} scope - controls the permissions that the new token will have
+   * @param {TokenScope} scope - controls the permissions that the new key will have
    * @param {string} expiresIn - How long the token is valid for in epoch timestamp.
-   * @returns {Promise<GenerateAuthToken.Response>} -
-   * {@link GenerateAuthToken.Success} containing the api token, refresh token, origin and epoch timestamp when token expires.
+   * @returns {Promise<GenerateApiKey.Response>} -
+   * {@link GenerateApiKey.Success} containing the api key, refresh token, origin and epoch timestamp when token expires.
    * If the token never expires, then no refresh token will be returned and expires at timestamp will be infinite.
-   * {@link GenerateAuthToken.Error} on failure.
+   * {@link GenerateApiKey.Error} on failure.
+   */
+  public async generateApiKey(
+    scope: TokenScope,
+    expiresIn: ExpiresIn
+  ): Promise<GenerateApiKey.Response> {
+    return await this.authClient.generateApiKey(scope, expiresIn);
+  }
+
+  /**
+   * @deprecated please use `generateApiKey` instead
    */
   public async generateAuthToken(
     scope: TokenScope,
     expiresIn: ExpiresIn
-  ): Promise<GenerateAuthToken.Response> {
-    return await this.authClient.generateAuthToken(scope, expiresIn);
+  ): Promise<GenerateApiKey.Response> {
+    return await this.generateApiKey(scope, expiresIn);
   }
 
   /**
-   * Refreshes an auth token.  Returns a new set of refresh/auth tokens that will be able to be refreshed again in the future.
-   * The new auth token will be valid for the same length of time as the original token, starting from the time of refresh.
-   * The original api token will still work until its expired.
+   * Refreshes an API key.  Returns a new API key and refresh token, that will be able to be refreshed again in the future.
+   * The new API key will be valid for the same length of time as the original token, starting from the time of refresh.
```

Comment:
```suggestion
   * The new API key will be valid for the same length of time as the original api key, starting from the time of refresh.
```

Reply:
fixed in https://github.com/momentohq/client-sdk-javascript/pull/838/commits/f85c3a7deb1fdb69f483f74caf7e5bc9f852dd36 , thanks!